README.rst

```diff
@@ -2,7 +2,10 @@ FirebaseArduino is a library to simplify connecting to the Firebase database fro
 arduino clients.
 
 It is a full abstraction of Firebase's REST API exposed through C++ calls in a wiring
-friendly way. All Json parsing is handled by the library and you may deal in pure C/Arduino
-types.
+friendly way.
+
+ArduinoJson is no longer part of this library and you will have to install latest version
```

Comment:
Maybe we should add a new `# DEPENDENCIES` sections in the README?

Reply:
Done.